src/psse_export.jl

```diff
@@ -0,0 +1,1343 @@
+        push!(my_blocked, id)
+    end
+    if singles_to_1
+        for (name, container_id) in zip(component_names, container_ids)
+            (length(ids_by_container[container_id]) == 1) &&
+                (id_mapping[(container_id, name)] = "1")
+        end
+    end
+    return id_mapping
+end
+
+"Take the output of `create_component_ids` and make it more suitable for JSON serialization"
+serialize_component_ids(id_mapping::Dict{Tuple{Int64, String}, String}) =
+    Dict("$(s_bus_n)_$(s_name)" => p_name for ((s_bus_n, s_name), p_name) in id_mapping)
+serialize_component_ids(id_mapping::Dict{Tuple{Tuple{Int64, Int64}, String}, String}) =
+    Dict(
+        "$(s_bus_1)-$(s_bus_2)_$(s_name)" => p_name for
+        (((s_bus_1, s_bus_2), s_name), p_name) in id_mapping
+    )
+
+# Fetch PL, QL, IP, IQ, YP, YQ
+_psse_get_load_data(exporter::PSSEExporter, load::PSY.StandardLoad) =
+    with_units_base(exporter.system, PSY.UnitSystem.NATURAL_UNITS) do
+        PSY.get_constant_active_power(load),
+        PSY.get_constant_reactive_power(load),
+        PSY.get_current_active_power(load),
+        PSY.get_current_reactive_power(load),
+        PSY.get_impedance_active_power(load),
+        PSY.get_impedance_reactive_power(load)
+    end
+
+# Fallback if not all the data is available
+# This mapping corresponds to `function make_power_load` in the parser
+_psse_get_load_data(exporter::PSSEExporter, load::PSY.StaticLoad) =
+    with_units_base(exporter.system, PSY.UnitSystem.NATURAL_UNITS) do
+        PSY.get_active_power(load),
+        PSY.get_reactive_power(load),
+        PSSE_DEFAULT,
+        PSSE_DEFAULT,
+        PSSE_DEFAULT,
+        PSSE_DEFAULT
+    end
+
+"""
+WRITTEN TO SPEC: PSS/E 33.3 POM 5.2.1 Load Data
+"""
+function write_to_buffers!(
+    exporter::PSSEExporter,
+    ::Val{Symbol("Load Data")},
+)
+    io = exporter.raw_buffer
+    md = exporter.md_dict
+    check_33(exporter)
+
+    loads = get!(exporter.components_cache, "loads") do
+        sort!(collect(PSY.get_components(PSY.StaticLoad, exporter.system)); by = PSY.get_name)
+    end
+    load_name_mapping = get!(exporter.components_cache, "load_name_mapping") do
+        create_component_ids(
+            convert_empty_stringvec(PSY.get_name.(loads)),
+            PSY.get_number.(PSY.get_bus.(loads));
+            singles_to_1 = true,
+        )
+    end
+    for load in loads
+        sienna_bus_number = PSY.get_number(PSY.get_bus(load))
+        I = md["bus_number_mapping"][sienna_bus_number]
+        ID = _psse_quote_string(load_name_mapping[(sienna_bus_number, PSY.get_name(load))])  # TODO should this be quoted?
+        STATUS = PSY.get_available(load) ? 1 : 0
+        AREA = PSSE_DEFAULT  # defaults to bus's area
+        ZONE = PSSE_DEFAULT  # defaults to zone's area
+        PL, QL, IP, IQ, YP, YQ = _psse_get_load_data(exporter, load)
+        OWNER = PSSE_DEFAULT  # defaults to bus's owner
+        SCALE = PSSE_DEFAULT  # TODO reconsider
```

Comment:
@jd-lara should our loads be marked as scalable?

Reply:
no, we don't need that for our exporter/models